test/e2e/network/service.go

```diff
@@ -2454,6 +2454,8 @@ func createPodOrFail(c clientset.Interface, ns, name string, labels map[string]s
 
 	_, err := c.CoreV1().Pods(ns).Create(context.TODO(), pod, metav1.CreateOptions{})
 	framework.ExpectNoError(err, "failed to create pod %s in namespace %s", name, ns)
+	err = e2epod.WaitTimeoutForPodReadyInNamespace(c, name, ns, framework.PodStartTimeout)
```

Comment:
Don't we want to use CreateSync() as you use in other places?

Reply:
I wanted to do that, but I need to change the function signature to pass the podClient() or the framework variable to call CreateSync() so I took the conservative option

```
test/e2e/network/loadbalancer.go:               hostExec := launchHostExecPod(f.ClientSet, f.Namespace.Name, "ilb-host-exec")
test/e2e/network/service.go:    hostExecPod := launchHostExecPod(c, ns, "verify-service-up-host-exec-pod")
test/e2e/network/service.go:    hostExecPod := launchHostExecPod(c, ns, "verify-service-down-host-exec-pod")
test/e2e/network/service.go:            hostExec := launchHostExecPod(f.ClientSet, f.Namespace.Name, "hostexec")
test/e2e/network/service.go:// launchHostExecPod launches a hostexec pod in the given namespace and waits
test/e2e/network/service.go:func launchHostExecPod(client clientset.Interface, ns, name string) *v1.Pod {
```

whatever do you think is better

Reply:
OK - I'm fine with doing that now, and cleaning this up later.